tests/wasmbus.rs

```diff
@@ -226,15 +231,18 @@ async fn wasmbus() -> anyhow::Result<()> {
         ("hostcore.osfamily".into(), FAMILY.into()),
         ("path".into(), "test-path".into()),
     ]);
+
     let (host, shutdown) = Host::new(HostConfig {
-        url: nats_url.clone(),
-        cluster_seed: Some(cluster_key.seed().unwrap()),
-        host_seed: Some(host_key.seed().unwrap()),
+        nats_host: nats_host.to_string(),
+        nats_port,
+        lattice_prefix: TEST_PREFIX.to_string(),
+        cluster_seed: cluster_key.seed().ok(),
+        host_seed: host_key.seed().ok(),
```

Comment:
Can we revert this change? This should panic if the seed key is missing and not silently ignore it - it would introduce a very hard-to-debug issue if any of these keys was not a seed key